.travis/travis-before.sh

```diff
@@ -5,6 +5,7 @@ sudo apt-get install -y nodejs
 sudo npm install -g ajv-cli
 sudo apt-get install shunit2
 docker pull repairnator/pipeline
+docker pull ycaxgjd/sequencer:1.0
```

Comment:
push to docker pull repairnator/sequencer:1.0

Reply:
I gave you the rights on DockerHub

Reply:
> I gave you the rights on DockerHub

Hey @martinezmatias Could you please check my permissions?
As when I push, I always meet `denied: requested access to the resource is denied`.

Reply:
Hey @monperrus 
Maybe I only have the read permission.